plugin-tester-java/src/main/java/example/myapp/statefulhelloworld/GreeterServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package example.myapp.statefulhelloworld;
+
+import example.myapp.statefulhelloworld.grpc.*;
+
+import akka.actor.ActorSystem;
+import akka.actor.ActorRef;
+import akka.util.Timeout;
+import static akka.pattern.PatternsCS.ask;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+// #stateful-service
+public final class GreeterServiceImpl implements GreeterService {
+
+  private final ActorSystem system;
+  private final ActorRef greeterActor;
+  private final Timeout timeout = Timeout.create(Duration.ofSeconds(5));
+
+  public GreeterServiceImpl(ActorSystem system) {
+    this.system = system;
+    this.greeterActor = system.actorOf(GreeterActor.props("Hello"), "greeter");
+  }
+
+  public CompletionStage<HelloReply> sayHello(HelloRequest in) {
+    return ask(greeterActor, GreeterActor.GET_GREETING, timeout)
+        .thenApply(message ->
+          HelloReply.newBuilder()
+            .setMessage(((GreeterActor.Greeting) message).greeting)
```

Comment:
Do we have a `mapTo` in scaladsl?

Reply:
`mapTo` is only for Scala as far as I could see (probably since it cannot be an extension method on `CS` like it is in Scala)